editor/editor_node.cpp

```diff
@@ -5883,7 +5877,7 @@ Variant EditorNode::drag_files_and_dirs(const Vector<String> &p_paths, Control *
 		Label *label = memnew(Label);
 
 		if (p_paths[i].ends_with("/")) {
-			label->set_text(p_paths[i].substr(0, p_paths[i].length() - 1).get_file());
+			label->set_text(p_paths[i].left(-1).get_file());
```

Comment:
You have a few of these where you kept `substr` and a few where you changed them for `left`. Your PR summary says that `left` should be used with positive length. Though in code negative is also supported.

Reply:
`substr()` with -1 gives the rest of the string, while left with -1 gives the string without the last character. So `left(-1)` is supported, but would be a behavior change.

I think I got down everything where behavior wouldn't change.